FIRO_TSEnsembles/src/main/java/hec/SqliteDatabase.java

```diff
@@ -676,12 +676,12 @@ private MetricCollection getMetricCollection(ResultSet rs) throws SQLException {
         byte[] byte_value_array = rs.getBytes("byte_value_array");
 
         float[][] values = TableCompression.UnPack(byte_value_array, member_count, member_length, compression);
-        String[] params = metricstats.split(",");
-        Statistics[] stats = new Statistics[params.length];
-        for (int i = 0; i < params.length; i++){
-            stats[i] = Statistics.valueOf(params[i]);
-        }
-        return new MetricCollection(issue_date, start_date, stats, values); //Duration.ofSeconds(interval_seconds),units);
+//        String[] params = metricstats.split(",");
```

Comment:
Would be nice if the commented lines are removed.

Reply:
done